README.md

```diff
@@ -214,38 +215,48 @@ This code can automatically convert an OpenSim model to the external function us
 
 
 #### S.post_process
 
-- **S.post_process.make_plot**: 
-	- boolean to plot post processing results (0 or 1). Default is *0*.
 - **S.post_process.rerun**: 
-	- boolean to rerun post-processing without solving OCP (0 or 1). Default is *0*. If this option is set to 1, one should specify the S.post_process.result_filename.
-- **S.post_process.result_filename**: 
-	- File name for results. Used for the name of .mat file that saves the results, diary of the OCP, and name of the .mot file of the output motion. When rerunning post-processing of an existing result, giving this file name is required. Default value is 
-- **S.post_process.savename**: 
-	- Type of savename to use if S.post_process.result_filename is empty. Defaults is *structured* [char]. This uses the name of the .mat file of results is used as <S.subject.name>_v\<n>. Where <S.subject.name> is defined in S.subject.name. n = 1 if <S.subject.name>_v1.mat does not exist. n is increased until n is found such that <S.subject.name>_v\<n>.mat does not exist. To change this structuring process, change its implementation in [run_pred_sim.m file](./run_pred_sim.m). An alternative option is *datetime* [char], this uses <S.subject.name>\_\<yyyymmddTHHMMSS>. Where \<yyyymmddTHHMMSS> is the system date and time when creating the savename.
+	- boolean to rerun post-processing without solving OCP (0 or 1). Default is *0*. If this option is set to 1, one should specify the S.misc.result_filename.
+- **S.post_process.load_prev_opti_vars**:
+	- load w_opt and reconstruct R before rerunning the post-processing. Advanced feature, for debugging only, you should not need this.
 
 #### S.solver
 
-- **S.solver.linear_solver**: 
-	- solver algorithm used for the OCP. Default is *mumps* [char].
-- **S.solver.tol_ipopt**: 
-	- the power (10^-x) the tolerance ipopt has to reach before the OCP can be regarded as solved; a higher number gives a more precise answer. Default is *4* [double]
-- **S.solver.max_iter**: 
-	- maximal amount of iterations after wich the solver will stop. Default is *10000* [double]
+- **S.solver.CasADi_path**:
+	- Path to CasADi installation (top folder). By default, this will use the CasADi installation that is in the matlab search path.
+- **S.solver.N_meshes**: 
+	- number of mesh intervals. Default is *50* [double] for S.misc.gaitmotion_type = HalfGaitCycle and *100* for FullGaitCycle	
+- **S.solver.run_as_batch_job**: 
+	- specify if the OCP is to be solved as a batch job. Default is *false* [bool]. Batch processing requires the [Parallel Computing Toolbox](https://nl.mathworks.com/products/parallel-computing.html).
 - **S.solver.parallel_mode**: 
 	- type of parallel computing. Default is *thread* [char].
```

Comment:
For settings like this, wouldn't it be helpful to give the other options along with the default?